core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/dot-edit-content-category-field.component.ts

```diff
@@ -71,18 +76,25 @@ export class DotEditContentCategoryFieldComponent {
      */
     field = input.required<DotCMSContentTypeField>();
 
-    // TODO: Replace with the content of the selected categories
-    values = [];
-    readonly #destroyRef = inject(DestroyRef);
-    #componentRef: ComponentRef<DotEditContentCategoryFieldSidebarComponent>;
+    /**
+     * Represents a DotCMS contentlet.
+     *
+     */
+    contentlet = input.required<DotCMSContentlet>();
+
+    readonly store = inject(CategoryFieldStore);
 
     /**
-     * Checks if the object has categories.
-     * @returns {boolean} - True if the object has categories, false otherwise.
+     * Determines if there are any selected categories.
+     *
+     * @returns {Boolean} - True if there are selected categories, false otherwise.
      */
-    hasCategories(): boolean {
-        return this.values.length > 0;
-    }
+    hasSelectedCategories = computed(() => {
```

Comment:
@oidacra Why should this computed not be in the store using `withComputed`?

Reply:
let me try, I think yes, I will move it.

Reply:
Changed to use the store, but with a method (I cant use direct the store in the host property)